### PR TITLE
Fix delivery price calculation when two packages have the same name

### PR DESCRIPTION
**Pull request by Atala**

### CODE REVIEW

File: src/Entity/PackageSet.php [hunk 1]
```diff
@@ -40,6 +41,13 @@ class PackageSet
 
     protected $id;
     protected $name;
+
+    /**
+     * @Assert\Unique(
+     *  normalizer="AppBundle\Entity\Package::getPackageName",
```
Comment (vladimir-8) [inline src/Entity/PackageSet.php:47]: Do you do it because there is no way to call `getName` directly?

File: features/retail_prices.feature
Comment (vladimir-8) [inline features/retail_prices.feature:1]: 👍

File: src/Entity/Task.php [hunk 2]
```diff
@@ -1401,4 +1401,13 @@ public function setDistanceFromPrevious($distanceFromPrevious)
...
     }
+
+    public function getStore(): Store|null
+    {
+        if(!is_null($this->getDelivery())) {
+            return $this->getDelivery()->getStore();
+        }
+
+        return null;
+    }
```
Comment (vladimir-8) [inline src/Entity/Task.php:1412]: is not it the same as `$this->getDelivery()?->getStore()`

**Review by vladimir-8 — APPROVED** [review]

File: src/Entity/Task.php (hunk 2 above)
Comment (Atala) [reply src/Entity/Task.php:1412]: ah yes indeed i didnt know this PHP syntax

File: src/Entity/PackageSet.php (hunk 1 above)
Comment (Atala) [reply src/Entity/PackageSet.php:47]: I dont know exactly. "normalizer" arg has to be an anonymous function so I had difficulties to find the correct format so I ended up with a static function.

File: src/Entity/Task.php (hunk 2 above)
Comment (Atala) [reply src/Entity/Task.php:1412]: I will change

**Pull request merged**